_docs/example-collections.md

```diff
@@ -926,19 +926,31 @@ headers.Add("Authorization-Key", "<key from the webhook headers>");
 if (configuration.ValidWebhookRequest(url, webhookContent, headers))
 {
     Webhook webhook = configuration.ParseString<Webhook>(webhookContent);
-    if (webhook.Event.StartsWith("transaction")) {
-        TransactionWebhook transactionWebhook = configuration.ParseString<TransactionWebhook>(webhookContent);
-        Guid transactionId = transactionWebhook.Object.Id;
-        string externalId = transactionWebhook.Object.ExternalId;
-        if (webhook.Event == "transaction.paid_in") {
```

Comment:
I don't think we want to replace all the transaction webhooks in the documentation with only payin method webhooks?